tox.ini

```diff
@@ -10,9 +10,9 @@ skip_missing_interpreters=True
 [testenv]
 commands =
 	{envpython} setup.py install
-	{envpython} pycodestyle.py --testsuite testsuite
+	{envpython} pycodestyle.py --max-doc-length=79 --testsuite testsuite
```

Comment:
These should probably also be 72, no?

Reply:
If I change this to 72, I need to actually fix a bunch of docstrings in pycodestyle itself (I can do that)

Reply:
I think that is the right way to go, that we should eat our own dog food, as it were. :)